src/Kanbas/Courses/Quizzes/Editor/index.css

```diff
@@ -0,0 +1,7 @@
+.wd-tabs .nav-item {
+    color: green;
```

Comment:
Where is this green used? I don't see it anywhere.

Reply:
Deleted